RUNNING_TESTS.md

```diff
@@ -12,24 +12,18 @@ It is possible to use [Visual Studio Community edition](https://www.visualstudio
 
 ## Building
 
-### Copy `Local.props.example` and Edit
+Simply run
 
-First copy `Local.props.example` to `Local.props` and edit it if needed, specifying whether
-you intend to build the client on Mono and what .NET version should be targeted.
+    build.sh|bat
```

Comment:
I'd simply make it say `build.bat` and `When running on Mono, use build.sh` on the next line.